Pandas 3 Transiton in Release v26.08 (#773)
rapidsai/docs: _notices/rsn0059.md

---
layout: notice
parent: RAPIDS Support Notices
grand_parent: RAPIDS Notices
nav_exclude: true
notice_type: rsn
# Update meta-data for notice
notice_id: 59 # should match notice number
notice_pin: true # set to true to pin to notice page

> title: "RAPIDS 26.08 release, cudf will transition to Panda 3.0+"

Reviewer: ```suggestion
title: "RAPIDS 26.08 release, cuDF will transition to Panda 3.0+"
```

> title: "RAPIDS 26.08 release, cudf will transition to Panda 3.0+"

Reviewer 2: In addition to James' suggestion, also need
```suggestion
title: "RAPIDS 26.08 release, cudf will transition to Pandas 3.0+"
```

Reviewer: applied via Prem's suggestion: https://github.com/rapidsai/docs/pull/773#discussion_r3267084336

> title: "RAPIDS 26.08 release, cuDF will transition to Panda 3.0+"

Reviewer 3: ```suggestion
title: "RAPIDS 26.08 release, cuDF will transition to Pandas 3.0+"
```

notice_author: RAPIDS TPM
notice_status: In Progress
notice_status_color: yellow
# 'notice_status' and 'notice_status_color' combinations:
#   "Proposal" - "blue"
#   "Completed" - "green"
#   "Review" - "purple"
#   "In Progress" - "yellow"
#   "Closed" - "red"
notice_topic: Platform Support Change
notice_rapids_version: "v26.08+"
notice_created: 2026-05-06
# 'notice_updated' should match 'notice_created' until an update is made

> notice_updated: 2026-05-06

Reviewer: ```suggestion
notice_created: 2026-05-19
# 'notice_updated' should match 'notice_created' until an update is made
notice_updated: 2026-05-19
```

These have drifted a lot since the PR was opened.

---

## Overview

> Starting in RAPIDS `v26.08`release, scheduled for August 6, 2026 cudf will drop support for Pandas 2 and will upgrade to Pandas 3.0+

Reviewer: ```suggestion
Starting in RAPIDS `v26.08` release, scheduled for August 6, 2026 cuDF will drop support for Pandas 2 and will upgrade to Pandas 3.0+.
```

This should be either the product name ("cuDF") or inline markup with the library name (`cudf`). I vote product name.

> Starting in RAPIDS `v26.08`release, scheduled for August 6, 2026 cudf will drop support for Pandas 2 and will upgrade to Pandas 3.0+

Reviewer 2: Just to make it more explicit that it's a hard requirement.
```suggestion
Starting in RAPIDS `v26.08`release, scheduled for August 6, 2026 cudf will drop support for Pandas 2 and will require to Pandas 3.0+
```

Reviewer: fixed in https://github.com/rapidsai/docs/pull/773/commits/4759e3d91b06ccf9cead9403fe3e61980a20d914 (GitHub wouldn't let me apply your suggestion directly because the commit it was on is too outdated at this point)

> Starting in RAPIDS `v26.08` release, scheduled for August 6, 2026 cuDF will drop support for Pandas 2 and will upgrade to Pandas 3.0+.

Reviewer: ```suggestion
Starting in RAPIDS `v26.08` release, scheduled for August 6, 2026 cuDF will drop support for Pandas 2 and will require Pandas 3.0+.
```
